compel/include/uapi/infect-util.h

```diff
@@ -3,11 +3,20 @@
 
 #include "common/compiler.h"
 
+/**
+ * The length of the hash is based on what libuuid provides.
+ * According to the manpage this is:
+ *
+ * The uuid_unparse() function converts the supplied UUID uu from the binary
+ * representation into a 36-byte string (plus trailing '\0')
+ */
+#define RUN_ID_HASH_LENGTH 36+1
```

Comment:
brackets around 36+1 are required (37 should work too). Otherwise expressions like 2 * RUN_ID_HASH_LENGTH don't work as expected.

Reply:
done